### PR TITLE
Add cross compile support for x32 builds

### PR DESCRIPTION
**Pull request by wmborelli**

### CODE REVIEW

File: ubuntu16/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:xenial
 
-RUN apt-get update && apt-get install -y \
+RUN dpkg --add-architecture i386 \
```
Comment (wmborelli) [inline ubuntu16/Dockerfile:3]: Not sure if we want to split this into multiple RUN statements

Comment (pmfarnsw) [reply ubuntu16/Dockerfile:3]: I don't think it matters to much.  This seems fine and it will make the container have less layers.

**Review by pmfarnsw — APPROVED** [review]
Looks good to me.

**Review by pmfarnsw — APPROVED** [review]

File: ubuntu16/scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+set -e
+
+usage ()
+{
+    cat <<- _EOF_
+
+#########################################################################################
+ Options:
+ -h or --help              Display the HELP message and exit.
+ --release                 Release number of the build.  Default 1
```
Comment (jwstric2) [inline ubuntu16/scripts/build.sh]: in the rpmbuilder, i'll have to confirm, i think we decided to remove this logic out and let the callee set the version ahead of time.  it's fine for now though, only something to consider in future.

**Review by jwstric2 — APPROVED** [review]
looks fine to me, the one comment i left can be a consideration for later.

File: ubuntu16/scripts/build.sh (hunk 2 above)
Comment (wmborelli) [reply ubuntu16/scripts/build.sh]: That's actually dead code. I'll remove it and then merge.

**Comment by wmborelli** [comment]
resolves #2

**Pull request merged**